src/StudyEntities/Candidate/Sex.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+/**
+ * This defines valid values for the Sex data point for a Candidate.
+ * It is a Value Object used to assist with validation.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\StudyEntities\Candidate;
+
+/**
+ * This defines valid values for the Sex data point for a Candidate.
+ * It is a Value Object used to assist with validation.
+ *
+ * @category Main
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Sex
+{
+    /* @var string */
+    public $value;
+
+    private const VALID_VALUES = array(
```

Comment:
@johnsaigle I think LORIS has `Other` is a possibility for one of the valid values. I remember someone mentioned this in the Loris slack channel.

Reply:
That sort of rings a bell, but I can't find that option in the codebase. The database lists an enum of `Male`, `Female` so I went with that.

Reply:
@johnsaigle found what I vaguely remember here https://test-loris-dev.loris.ca/dicom_archive/?site=Data+Coordinating+Center&sex=O

Reply:
I opened an issue for it and added an item to the meeting agenda because these two should match but I'm not sure which is correct for projects.

Reply:
@maltheism I've added 'Other' as an option